envs/example/defaults.yml

```diff
@@ -107,8 +107,11 @@ nova:
     debug: True
     verbose: True
 
+ceph:
+  enabled: false
+
 cinder:
-  enabled: True
+  enabled: {{ ceph.enabled }}
   volume_type: file
```

Comment:
Yeah, we will need changes to that and `cinder.conf`. Does this look like the right approach? Also, did you see my question in Slack:

`Do we only want the cinder control plane and cinder data plane when ceph is enabled?`